### PR TITLE
Add action metrics for schedule and updates

### PR DESCRIPTION
**Pull request by yiminc**
**What changed?**
Add more action metrics for schedules and updates

**Why?**
Add more actions

**How did you test it?**
eye_balls

**Potential risks**
No

**Is hotfix candidate?**
No

### CODE REVIEW

File: common/rpc/interceptor/telemetry.go [hunk 1]
```diff
@@ -62,19 +62,24 @@ var (
 
 // static variables used to emit action metrics.
 var (
-	respondWorkflowTaskCompleted = "RespondWorkflowTaskCompleted"
+	respondWorkflowTaskCompleted = "RespondWorkflowTaskCompletedRespondWorkflowTaskCompleted"
```
Comment (stephanos) [inline common/rpc/interceptor/telemetry.go:65]: is `RespondWorkflowTaskCompleted` meant to be repeated?

File: common/rpc/interceptor/telemetry.go [hunk 2]
```diff
@@ -62,19 +62,24 @@ var (
...
+		"StartWorkflowExecution":           {},
+		"SignalWorkflowExecution":          {},
+		"SignalWithStartWorkflowExecution": {},
+		"RespondWorkflowTaskCompleted":     {},
+		"PollActivityTaskQueue":            {},
+		"CreateSchedule":                   {},
+		"UpdateSchedule":                   {},
+		"DeleteSchedule":                   {},
+		"PatchSchedule":                    {},
+		"UpdateWorkflowExecution":          {},
```
Comment (stephanos) [inline common/rpc/interceptor/telemetry.go:82]: What's the benefit of inlining the strings; couldn't we just use the `const`s? The benefit of that is they have (some) documentation.

File: common/rpc/interceptor/telemetry.go (hunk 1 above)
Comment (yiminc) [reply common/rpc/interceptor/telemetry.go]: No. Must be caused by some unintentional key strokes before I sent PR. :(

File: common/rpc/interceptor/telemetry.go (hunk 2 above)
Comment (yiminc) [reply common/rpc/interceptor/telemetry.go:82]: I guess it is easer to read with just the string literal, but maybe not. The old metrics.FrontendXxx const was used to tag metrics for each method (not related to actions here) before we moved that tag into auto gen. Now they are not used anywhere else, so I think it is better to remove them. If we want const, we can define private const here in this file.

**Review by tdeebswihart — APPROVED** [review]

**Pull request merged**